Source/Models/API/PHAPIEndpoint.swift

```diff
@@ -27,7 +27,7 @@ class PHAPIEndpoint {
         updateAuthorizationToken()
     }
 
-    func updateAuthorizationToken(token: PHToken? = PHKeychain.getToken()) {
+    func updateAuthorizationToken(token: PHToken? = store.state.token) {
```

Comment:
Can we always pass the token. Making `PHAPIEndpoint` depend on a global variable would be problematic.